bin/attacks/bruteforce/bf_attack.py

```diff
@@ -1,9 +1,9 @@
 from __future__ import print_function
 
-import itertools
 import os
 
 from bin.verify_hashes.verify import verify_hash_type
+from bin.generators import word_generator
```

Comment:
`bin` seems like an odd name for a directory since these are not stand alone executables.

> [/bin is a standard subdirectory of the root directory in Unix-like operating systems that contains the executable (i.e., ready to run) programs that must be available in order to attain minimal functionality for the purposes of booting (i.e., starting) and repairing a system.](http://www.linfo.org/bin.html)